collectd/files/write_graphite.conf

```diff
@@ -10,16 +10,18 @@
 LoadPlugin write_graphite
 
 <Plugin write_graphite>
- <Carbon>
-   Host "{{ collectd_settings.plugins.write_graphite.host }}"
-   Port "{{ collectd_settings.plugins.write_graphite.port }}"
-   Prefix "{{ collectd_settings.plugins.write_graphite.prefix }}."
-   Postfix "{{ collectd_settings.plugins.write_graphite.postfix }}"
-   Protocol "{{ collectd_settings.plugins.write_graphite.protocol }}"
-   EscapeCharacter "{{ collectd_settings.plugins.write_graphite.escapecharacter }}"
-   LogSendErrors {{ collectd_settings.plugins.write_graphite.logsenderrors|lower }}
-   SeparateInstances {{ collectd_settings.plugins.write_graphite.separateinstances|lower }}
-   StoreRates {{ collectd_settings.plugins.write_graphite.storerates|lower }}
-   AlwaysAppendDS {{ collectd_settings.plugins.write_graphite.alwaysappendds|lower }}
- </Carbon>
+{%- for node_name, graphite_node in collectd_settings.plugins.write_graphite.nodes.iteritems() %}
+ <Node "{{ node_name }}">
```

Comment:
The only thing I'm coming up with is a setting that goes in map.jinja. Default it to `new_graph_config: False` and then set it to True using another filter_by map that filters on lsb_codename (to differentiate the versions).

There are probably better ways.